### DISCUSSION TITLE
Contribution problem

### DISCUSSION THREAD

**Discussion by Viveksati5143** [community/community#34071] [Other Feature Feedback, Questions, & Ideas]
I pushed some codes using git bash and they are successfully pushed in the repo, but can someone tell me why a contibution is not being added in streak in my profile???????????????????????????

**Comment by Pralhad-Nasane** [comment]
I think this GitHub doc help you [Contribution-help](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#which-contributions-are-counted)
if still you are facing an issue let me know.

**Comment by james-villanueva-freee** [comment]
Hi, I remember having the same problem before when I have more than 1 branch in a repo. If I remember correctly, I had a (default) master branch and some working branch where I push my changes. The contributions will not show up until you merge the working branches to the default branch or the master branch.

In your case, I think the source of your problem is this repo [Viveksati5143/JavaBasicProgram](https://github.com/Viveksati5143/JavaBasicProgram).
It has 2 branches:
  1. a `master` branch which was updated a few months ago
  2. a `main` branch which has your latest updates.
  
To solve your problem, you have to create a pull request to _combine_ the changes to both branches.  Just click the compare button [here](https://github.com/Viveksati5143/JavaBasicProgram/branches) and proceed in creating that pull request. Don't forget to merge it after.

Also, here is a [stackoverflow thread](https://stackoverflow.com/questions/15289768/github-commits-arent-recorded-in-the-your-contributions-calendar) I found with a problem similar to yours.

Let me know if you need additional help. Cheers! 🙇🏼‍♂️

**Reply by airtower-luna** [reply]
Merging is one option, changing which branch is the default branch is another. It depends on what you want to do with your branches. And of course "both" is possible, too, e.g. if you want `main` to be the default branch but `master` has some changes that aren't on `main` yet.
